plugin/core/signature_help.py

```diff
@@ -66,26 +65,18 @@ def render(self, view: sublime.View) -> str:
         except IndexError:
             return ""
         formatted = []  # type: List[str]
-        intro = self._render_intro()
-        if intro:
-            formatted.append(intro)
+        if self.has_multiple_signatures():
+            formatted.append(self._render_intro())
         formatted.extend(self._render_label(view, signature))
         formatted.extend(self._render_docs(view, signature))
         return "".join(formatted)
 
-    def context(self, trigger_kind: int, trigger_character: str, is_retrigger: bool) -> SignatureHelpContext:
+    def active_signature_help(self) -> SignatureHelp:
```

Comment:
I named it after the "activeSignatureHelp" key of SignatureHelpContext, and the only purpose of this function right now is to retrieve this value in preparation of a new request. While it is true that it will update `self._state` (just because it is simpler this way than to create a whole new SignatureHelp object for the output value of this function), the state isn't used anymore and discarded as soon as the new response arrives. In general I'm not very picky in regard of naming and could adjust it if you want, but I don't think it would be very useful because this function isn't used elsewhere.